src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -170,6 +184,14 @@ export interface SavedObjectsIncrementCounterField {
   incrementBy?: number;
 }
 
+/**
+ * @internal
+ */
+export interface SavedObjectsFindInternalOptions {
+  /** This is used for internal consumers that need to use a PIT finder but want to prevent extensions from functioning. */
```

Comment:
Yes, I think what Joe means by internal consumer is functionality within the SO domain that is using the same functions but need some additional options that we don't want to expose otherwise.